spec/controllers/users_controller_spec.rb

```diff
@@ -279,7 +279,8 @@
         end
 
         it 'redirects back' do
-          expect { submit }.to redirect_back
+          submit
+          expect(response).to have_http_status :unprocessable_entity
```

Comment:
This should now be failing but... isn't? But anyways should be testing with `redirect_to`